src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -78,3 +79,36 @@ export const getTeamReputationChartData = (
 
   return topTeams;
 };
+
+export const getContributorReputationChartData = (
+  contributorsList: ContributorItem[],
+): ReputationChartDataItem[] => {
+  const topContributors = contributorsList
+    .slice(0, WIDGET_TEAM_LIMIT)
+    .map(({ walletAddress, user, reputation }, index) => {
+      return {
+        id: walletAddress,
+        label: user?.profile?.displayName || '',
+        value: Number(reputation),
+        color: getTeamHexColor(CONTRIBUTORS_COLORS_LIST[index]),
+      };
+    });
+
+  const reputationInOtherContributors = contributorsList
+    .slice(WIDGET_TEAM_LIMIT)
+    .reduce((reputation, team) => reputation + Number(team.reputation), 0);
```

Comment:
In order to avoid numbers not adding up to `100%`, I would suggest refactoring this

```suggestion
const reputationInOtherContributors =
    100 -
    topContributors.reduce(
      (reputation, topContributor) => reputation + topContributor.value,
      0,
    );
```

Reply:
There is a rounding calculation bug somewhere in Amplify or similar, so for now, this change will not be implemented. thank you 🤩